scripts/ci/lib.sh

```diff
@@ -31,7 +68,7 @@ push_images() {
         local tag="$2"
 
         for image in "${image_set[@]}"; do
-            "$ROOT/scripts/push-as-manifest-list.sh" "${registry}/${image}:${tag}" | cat
+            "$SCRIPTS_ROOT/scripts/push-as-manifest-list.sh" "${registry}/${image}:${tag}" | cat
```

Comment:
This pipe to cat seems redundant.

Reply:
I see; The pipe enforces stdout to not appear as a TTY for whatever is running inside `push-as-manifestt-Iist.sh`. Thanks.